src/libstd/fs/mod.rs

```diff
@@ -361,6 +436,15 @@ impl OpenOptions {
     ///
     /// This option indicates whether a new file will be created if the file
     /// does not yet already exist.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
Could you tag all these examples with `no_run`? It's been unfortunate in the past where doc tests start spraying files in the build dir :(